docker-compose.yaml

```diff
@@ -231,6 +160,7 @@ services:
       - NETWORK_NAME=cdm-jupyterhub-network
       - JUPYTERHUB_USER_IMAGE=cdm-jupyterhub-cdm_jupyterhub:latest
       - JUPYTERHUB_MOUNT_BASE_DIR=/cdr/cdm/jupyter
+      - ENVIRONMENT=dev
```

Comment:
Is there no user hub instance?

Reply:
no. Just one hub instance and permissions are controlled via user groups.

Reply:
What does the ENV variable do in that case?

Reply:
https://github.com/kbase/cdm-jupyterhub/blob/main/config/jupyterhub_config.py#L64-L66

Reply:
I'd probably give that env var a more descriptive name in that case, like REMOVE_CONTAINER or something